include/SDL_syswm.h

```diff
@@ -288,6 +219,7 @@ struct SDL_SysWMinfo
         Uint8 dummy[64];
```

Comment:
You might want to take the opportunity to make this larger? On Wayland you're already uncomfortably close to the size allowed (7 out of 8 pointers) so maybe expand it to 16 pointers.

I would personally size it something like

```
#define SDL_SYSWM_INFO_SIZE_V1 (sizeof (int) + sizeof (int) + 8 * (sizeof (void *) >= 8 ? sizeof (void *) : 8))

union
{
    ... all the genuinely useful stuff ...
    void *dummy_pointers[16];
    Uint64 dummy_ints[16];
} info;
```

so that it has room for at least 8 pointers or 64-bit integers, whichever is larger: people are starting to use architectures with 128-bit tagged pointers, so assuming pointers are <= 64 bits is looking less and less future-proof.